aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/BaseHandlerStd.java

```diff
@@ -106,7 +107,10 @@ public abstract class BaseHandlerStd extends BaseHandler<CallbackContext> {
             .build();
 
     protected final static HandlerConfig DB_INSTANCE_HANDLER_CONFIG_36H = HandlerConfig.builder()
-            .backoff(Constant.of().delay(Duration.ofSeconds(30)).timeout(Duration.ofHours(36)).build())
+            .backoff(Exponential.of()
+                    .minDelay(Duration.ofSeconds(30))
+                    .timeout(Duration.ofHours(36))
+                    .build())
```

Comment:
Is going to impact regular stabilization till instance reach available state afte creation/modification? If yes, Do you know duration of stabilization in happy case after this change.

Reply:
There would be handler invocations at the following time deltas after create or update request:

```
0:00:30
0:01:30
0:03:30
0:07:30
0:15:30
0:31:30
1:03:30
2:07:30
4:15:30
8:31:30
17:03:30
1 day, 10:07:30
```

Dependent on resource, parameters and the region, stabilization takes anywhere from 2 minutes and 30 minutes. I think happy case is covered pretty well.